PIL/Image.py

```diff
@@ -1236,6 +1307,7 @@ def getpalette(self):
             return None  # no palette
 
     def getpixel(self, xy):
+        # type: (Coord) -> Tuple[Any]
```

Comment:
Could this `Any` be constrained to a type of `float` or `int`? For example, `Tuple[Union[float, int], ...]`.